lib/logstash/outputs/amazon_es/http_client.rb

```diff
@@ -54,6 +41,7 @@ def initialize(options={})
       @metric = options[:metric]
       @bulk_request_metrics = @metric.namespace(:bulk_requests)
       @bulk_response_metrics = @bulk_request_metrics.namespace(:responses)
+      @target_bulk_bytes = options[:max_bulk_bytes] 
```

Comment:
Do we want make the naming consistent here?  Meaning change the target_bulk_bytes to max_bulk_bytes, which seems to be a better name to me or should we keep it as target_bulk_bytes to be consistent with the ES plugin?

Reply:
Will make this change